app/models/miq_automation_worker.rb

```diff
@@ -0,0 +1,13 @@
+class MiqAutomationWorker < MiqQueueWorkerBase
+  include MiqWorker::ReplicaPerWorker
+
+  require_nested :Runner
+
+  self.required_roles               = ["automate"]
+  self.default_queue_name           = "automate"
```

Comment:
I thought automate was run off the generic worker.
(Multiple automate workers running at the same time and have to be in the same zone/machine?)

But we are running as a single worker per zone (region?)

Reply:
MIQ automate does run on generic workers.  This is for Workflows / Ansible Runner

Reply:
This is an interesting point - wonder if we should put in a code comment?  Alternatively, what if we had different queue names for workflows and ansible runner stuff? 🤔

Reply:
Hm we'd have to do something interesting to get a worker to dequeue from multiple queue names but that isn't impossible

Reply:
please have a worker only listen to one queue. Don't think it buys us much overriding code like that.